front/public/locales/fr/operationalStudies/scenario.json

```diff
@@ -35,6 +35,7 @@
     "delete": "Supprimer",
     "duplicate": "Dupliquer",
     "importTrainSchedule": "Importer",
+    "invalidTrains": "La grille horaires comporte des trains invalides.",
```

Comment:
You forgot the English translation :)

Reply:
Done :)

Reply:
For the text in the popover, it has been decided during refinement and talks with operational studies to use the back error messages for now